app/jobs/importing/hud_zip/fetch_and_import_job.rb

```diff
@@ -7,22 +7,65 @@
 module Importing::HudZip
   class FetchAndImportJob < BaseJob
     queue_as ENV.fetch('DJ_LONG_QUEUE_NAME', :long_running)
+    WAIT_MINUTES = 15
 
-    def perform(klass:, options:)
-      safe_klass = known_classes.detect { |m| klass == m }
-      raise "Unknown import class: #{klass}; You must add it to the whitelist in FetchAndImportJob" unless safe_klass.present?
+    attr_accessor :job
 
-      safe_klass.constantize.new(**options).import!
+    def initialize(klass:, options:)
+      @klass = klass
+      @options = options
```

Comment:
I am a little confused as to why the params go into initialize instead of perform. I guess it's because we are enqueuing it from `Delayed::Job.enqueue` instead of ActiveJob? If we want to do it this way, should probably include a call to `super()` in initialize.

Also are we enqueuing it this way so we get access to `job` in the before hook?